packages/shopify-helpers/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @bsmnt/shopify-helpers
 
+## 2.0.0
+
+### Major Changes
+
+- 687fd41: Remove createStandardStorefrontClient as it was too big with no real benefit for now
```

Comment:
```suggestion
- 687fd41: Remove `createStandardStorefrontClient` as it was too big with no real benefit for now. Instead of using `createStandardStorefrontClient`, you now should use [`@bsmnt/sdk-gen`](https://github.com/basementstudio/commerce-toolkit#bsmntsdk-gen)
```